eodag/resources/providers.yml

```diff
@@ -1799,7 +1799,9 @@
     S2_MSI_L1C:
       productType: sentinel-2-l1c
       metadata_mapping:
-        id: '{$.properties."s2:product_uri"#remove_extension}'
+        id:
+          - id
```

Comment:
this doesn't work, please see how to configure query-string just bellow (`utmZone` for instance) and test if you can search by id using the updated configuration